base/sysinfo.jl

```diff
@@ -183,19 +183,19 @@ maxrss() = ccall(:jl_maxrss, Csize_t, ())
 if is_windows()
     function windows_version()
         verinfo = ccall(:GetVersion, UInt32, ())
-        (Int(verinfo & 0xFF), Int((verinfo >> 8) & 0xFF))
+        VersionNumber(verinfo & 0xFF, (verinfo >> 8) & 0xFF, verinfo >> 16)
     end
 else
-    windows_version() = (0, 0)
+    windows_version() = v"0.0"
 end
 """
     windows_version()
 
-Returns the version number for the Windows NT Kernel as a (major, minor) pair,
-or `(0, 0)` if this is not running on Windows.
+Returns the version number for the Windows NT Kernel as a VersionNumber,
+i.e. v"major.minor.build", or `v"0.0.0` if this is not running on Windows.
```

Comment:
missing a close quote on the 0, and code highlighting on the VersionNumber and major.minor example